login.router.js

```diff
@@ -0,0 +1,77 @@
+const mongoose = require('mongoose')
+const express  = require('express')
+const bodyParser = require('body-parser')
+const cors = require('cors')
+
+const app = express()
+app.use(bodyParser.json())
+app.use(cors())
+
+
+const uri = `${process.env['uri']}`;
+
+mongoose.connect(uri,{
+useUnifiedTopology:true,
+useNewUrlParser:true
+})
+
+const UserSchema = new mongoose.Schema({
+fullname : String,
+username : String,
+email : String,
+password : String,
+})
+
+const User = mongoose.model('User' , UserSchema)
+
+
+app.get("/", async (req,res)=>{
+  try{
+    const users = await User.find({});
+    res.json({success : true , users})
+  }catch (err) {
+      res.status(500).json({ success: false, message: "unable to get users", errorMessage: err.message })
+    }
+
+})
+
+app.post("/",async(req,res)=>{
```

Comment:
Over here you can use the express router to structure the code better.
You can chain multiple methods with the same path together.

For Example: 

```
const router = express.Router();

router.route("/")
.get((req,res) => {
 write the method over here
})
.post((req,res) => {
 write the method over here
})
```